src/app/tests/suites/certification/Test_TC_DRLK_2_1.yaml

```diff
@@ -1551,20 +1551,288 @@ tests:
       response:
           value: NumberOfCredentialsSupportedPerUserValue
 
-    - label: "Cleanup the created user"
-      PICS: DRLK.S.F00 && DRLK.S.F07
+    - label: "Step 36: TH sends ClearCredential Command to DUT"
+      PICS: DRLK.S.F00 && DRLK.S.F07 && DRLK.S.C26.Rsp
+      command: "ClearCredential"
+      timedInteractionTimeoutMs: 1000
+      arguments:
+          values:
+              - name: "Credential"
+                value: { CredentialType: 1, CredentialIndex: 1 }
+
+    - label:
+          "Step 37: TH sends ClearUser Command to DUT with the UserIndex as 1"
+      PICS: DRLK.S.F00 && DRLK.S.F07 && DRLK.S.C1d.Rsp
```

Comment:
Test plan PICS condition only has `DRLK.S.F08(USR), DRLK.S.C1d.Rsp(ClearUser)`. `DRLK.S.F00` is not part of the step.